app/Resources/CraueConfigBundle/translations/CraueConfigBundle.it.yml

```diff
@@ -38,3 +38,4 @@ demo_mode_username: "Utente Demo"
 api_user_registration: Abilita la registrazione dell'utente attraverso l'API
 # store_article_headers: Enable if wallabag stores HTTP headers for each article
 # shaarli_share_origin_url: Enable sharing origin url to Shaarli, if the service is enabled
+settings_changed: Configurazione aggiornata
```

Comment:
We should then remove all commented lines we left for translator then. Because weblate re-add the key when the translation is done.
`settings_changed` is commented on line 1.

Reply:
Weblate should be able to do that itself after the next Italian language edition.